salt/covid19.sls

```diff
@@ -1,5 +1,14 @@
-{% from 'lib.sls' import set_firewall %}
+{% from 'lib.sls' import set_firewall,create_user %}
```

Comment:
Add space after comma, follow same order as isort in Python.

```suggestion
{% from 'lib.sls' import create_user, set_firewall %}
```

Reply:
added space after comma